src/modules/interfaces.ts

```diff
@@ -539,6 +539,16 @@ export class ModuleBase {
       } catch (e) {
         logger.warn(`Unable to read file ${this.dirname}/${this.sql.afterInstallSqlPath}`);
       }
+    } else {
+      // If no path specified, try to get the default
+      try {
+        const sql = fs.readFileSync(`${this.dirname}/sql/after_install.sql`, 'utf8');
+        this.migrations.afterInstall = async (q: QueryRunner) => {
+          await q.query(sql);
+        };
+      } catch (e) {
+        logger.warn(`Unable to read default file ${this.dirname}/sql/after_install.sql`);
```

Comment:
I don't know if it should log anything in this case, since there may not even be a file in question for the module?